src/systems/filecoin_mining/storage_mining/_index.md

```diff
@@ -54,7 +53,7 @@ A particular sector enters `TemporaryFault` from `Active` through `DeclareTempor
 
 `DetectedFault` is a miner-wide PoSt state when all sectors are considered inactive. All power is lost immediately and pledge collateral is slashed. If a miner remains in `DetectedFault` for more than MaxConsecutiveFailures, all sectors will be terminated, both power and market actors will be notified.
 
-`ProvingSet` is consist of sectors that miners are required to generate proofs against and is what counts towards miners' power. In other words, `ProvingSet` is a set of all `Active` sectors for a particular miner. `ProvingSet` is only relevant when the miner is in OK stage of its `MinerPoStState`. When a miner is in the `Challenged` state, `ChallengedSectors` specify the list of sectors to be challenged which is the `ProvingSet` before the challenge is issued thus allowing more sectors to be added while it is in the `Challenged` state.
+`ProvingSet` consists of sectors that miners are required to generate proofs against and is what counts towards miners' power. In other words, `ProvingSet` is a set of all `Active` sectors for a particular miner. `ProvingSet` is only relevant when the miner is in OK stage of its `MinerPoStState`. When a miner is in the `Challenged` state, `ChallengedSectors` specify the list of sectors to be challenged which is the `ProvingSet` before the challenge is issued thus allowing more sectors to be added while it is in the `Challenged` state.
```

Comment:
maybe worth explicitly saying but
> sectors cannot be removed from `ProvingSet` when `Challenged`

Reply:
the list of Challenged sectors specified in Challenged state is copy of the ProvingSet at the time of challenge so hence it can't be added to or removed from.

Reply:
Then the above sentence is slightly confusing, esp
> allowing more sectors to be added to it...